src/backend/cuda/device_manager.cpp

```diff
@@ -396,7 +398,7 @@ DeviceManager::DeviceManager()
         if (i < nDevices) {
             auto prop = make_pair(cuDevices[i].prop.major,
                     cuDevices[i].prop.minor);
-            findJitDevCompute(prop);
+            checkAndSetDevMaxCompute(prop);
```

Comment:
Can you make sure its more resilient to future toolkit updates?

Reply:
Please check now